depends/packages/native_libtapi.mk

```diff
@@ -6,7 +6,7 @@ $(package)_sha256_hash=d4d46c64622f13d6938cecf989046d9561011bb59e8ee835f8f39825d
 $(package)_patches=disable_zlib.patch
 
 ifeq ($(strip $(FORCE_USE_SYSTEM_CLANG)),)
-$(package)_dependencies=native_clang
```

Comment:
e65a429b6a895d183aa2131b2001dfee0ec78240 delete `native_clang.mk`?

Reply:
It was, seems it returned in a rebase. Will drop it on the next push.